.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,72 @@
+name: "🐛 Bug report"
+description: Report for reproducible bug of cli-kintone
+labels: ["bug"]
+title: "[Bug] "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting a bug! Please fill out the followings.
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      description: Please describe what the bug is.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Target Version
+      placeholder: e.g. vX.Y.Z
+      description: Which version of cli-kintone do you want to report a bug for? 
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: |
+        Please provide [minimum reproduction](https://stackoverflow.com/help/minimal-reproducible-example) steps of the bug.
+        If you have a reproduction repository, [GitHub Gist](https://gist.github.com/) snippet, or some external pages, please share the links.
+        WARNING: DO NOT SUBMIT CONFIDENTIAL INFORMATION SUCH AS YOUR SUBDOMAIN AND PASSWORD
```

Comment:
@mshrtsr 

I think the warning message is the most important, so please consider moving it to the first line.

```
WARNING: DO NOT SUBMIT CONFIDENTIAL INFORMATION SUCH AS YOUR SUBDOMAIN AND PASSWORD

Please provide [minimum reproduction](https://stackoverflow.com/help/minimal-reproducible-example) steps of the bug.
If you have a reproduction repository, [GitHub Gist](https://gist.github.com/) snippet, or some external pages, please share the links.
```

Reply:
@hung-cybo 
Hmm... I think the most case, the note|caution|warning is after the main content.
I found some examples (not so many).
- https://docusaurus.io/docs/deployment#self-hosting
- https://jestjs.io/docs/api#testconcurrentname-fn-timeout

Reply:
Emojis get more attention than only text.
We decided to use ⚠️ as the prefix for our warning.